_news/2018-10-08-forseti-day-london.md

```diff
@@ -6,5 +6,4 @@ We’re pleased to invite you to Google’s Forseti Community Day on Friday,
 October 12th (Right after [Next London 2018](https://cloud.withgoogle.com/next18/london)
 , Next Ticket NOT required).
 Join us for a fun, interactive, and hands-on day shaping the next generation
-of Google Cloud Platform (GCP) security tools. [Register now to
-attend](https://docs.google.com/forms/d/1gnIZj9NqIv4_PskM1mItac9TSFDBO8LqdOuIYUZAMIE/edit).
+of Google Cloud Platform (GCP) security tools.
```

Comment:
Can you put a note there saying that the registration has been closed?

Reply:
Done